pkg/skaffold/jib/jib.go

```diff
@@ -17,22 +17,34 @@ limitations under the License.
 package jib
 
 import (
+	"fmt"
 	"os"
 	"os/exec"
+	"path/filepath"
 	"sort"
+	"strings"
 
 	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/util"
 	"github.com/karrick/godirwalk"
 	"github.com/pkg/errors"
 	"github.com/sirupsen/logrus"
 )
 
-func getDependencies(cmd *exec.Cmd) ([]string, error) {
+func getDependencies(workspace string, cmd *exec.Cmd) ([]string, error) {
 	stdout, err := util.RunCmdOut(cmd)
 	if err != nil {
 		return nil, err
 	}
 
+	// Jib's dependencies are absolute, and usually canonicalized, so must canonicalize the workspace
```

Comment:
I take it that "usually" means that they are not always canonicalized? Or do you simply want to cover that possibility `relativize`?

Reply:
To cover that possibility: the Jib `skaffold-files` goal simply reports builder-provided information.  So it's best not to assume anything, really.

Reply:
Frankly, I don't understand this comment yet either :) What do we exactly mean by "canonicalization"? Everything relative? Everything absolute? Symlinks resolved to absolute? Not sure I'm following.